components/service/glean/src/test/java/mozilla/components/service/glean/private/LabeledMetricTypeTest.kt

```diff
@@ -559,4 +559,44 @@ class LabeledMetricTypeTest {
         }
         assertEquals(1, snapshot.get("telemetry.labeled_metric/__other__"))
     }
+
+    @Test
+    fun `test recording to static labels by label index`() {
```

Comment:
I think we need a test for recording to dynamic labels by label index.

Reply:
We do not support indices for dynamic labels, sorry for the confusion. I clarified the comment above. Due to this, we don't need additional test coverage for dynamic labels: we already check for the case of "unknown" labels. We count accumulations to unknown labels in the `__other__` label.

Reply:
Okay. I've added a comment to https://bugzilla.mozilla.org/show_bug.cgi?id=1566355 to make sure we ensure the `labels` lengths match between Histograms.json and metrics.yaml.